src/gui/toolbarMenubar/ToolZoomSlider.cpp

```diff
@@ -1,182 +1,158 @@
 #include "ToolZoomSlider.h"
 
 #include <cmath>
-#include <utility>
-
-#include <config.h>
+#include <sstream>
 
 #include "control/zoom/ZoomControl.h"
 
-#include "StringUtils.h"
+#include "config.h"
 #include "i18n.h"
 
-ToolZoomSlider::ToolZoomSlider(ActionHandler* handler, std::string id, ActionType type, ZoomControl* zoom,
+constexpr double SCALE_LOG_OFFSET = 0.20753;
+
+constexpr double FINE_STEP_COUNT = 100.0;
+constexpr double COARSE_STEP_COUNT = 10.0;
+
+// TODO(personalizedrefrigerator): Update the range to reflect the min and max zoom settings.
+constexpr auto SLIDER_RANGE =
+        AbstractSliderItem::SliderRange{DEFAULT_ZOOM_MIN, DEFAULT_ZOOM_MAX, FINE_STEP_COUNT, COARSE_STEP_COUNT};
```

Comment:
Before this PR, the zoom slider's range was also from `DEFAULT_ZOOM_MIN` to `DEFAULT_ZOOM_MAX`. As such, while this should be updated to match the setting, this should not cause a regression.